### DISCUSSION TITLE
A way to preview the shared compose-ui?

### DISCUSSION THREAD

**Discussion by Nikola-Milovic** [badoo/Decompose#176] [Q&A]
Hello everyone, looking for a way to develop Compose Screens and have previews without having to have a dummy project at hand to use. I currently have a compose-ui module in the shared module, I share pretty much all of the Compose screens between the Desktop and Android, it seems that now it's possible to add @Preview annotation to the Desktop and the Android commons. Anyone has a sample at hand? I am having a hard time creating component stubs needed for the content (screens).

**Comment by arkivanov** [comment]
Thanks for pointing on that, I will add previous to all the sample apps! Filed #177.
At the moment you can check my Android Dev Challenge submissions. One of the examples is [here](https://github.com/arkivanov/android-dev-challenge-compose-1/blob/753201f65d7a00f0c6b9eedc3a06680d36eec162/app/src/main/java/com/example/androiddevchallenge/list/PuppyListUi.kt#L182).
